WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormInput.swift

```diff
@@ -2,13 +2,87 @@ import SwiftUI
 import Yosemite
 
 struct ShippingLabelCustomsFormInput: View {
+    private let isCollapsible: Bool
+    private let packageNumber: Int
+    private let safeAreaInsets: EdgeInsets
+
+    @ObservedObject private var viewModel: ShippingLabelCustomsFormInputViewModel
+    @State private var showingContentTypes = false
+
+    init(isCollapsible: Bool, packageNumber: Int, safeAreaInsets: EdgeInsets, viewModel: ShippingLabelCustomsFormInputViewModel) {
+        self.isCollapsible = isCollapsible
+        self.packageNumber = packageNumber
+        self.safeAreaInsets = safeAreaInsets
+        self.viewModel = viewModel
+    }
+
     var body: some View {
-        Text("Hello, World!")
+        CollapsibleView(isCollapsible: isCollapsible, safeAreaInsets: safeAreaInsets, label: {
+            headerView
+        }, content: {
+            VStack {
+                TitleAndToggleRow(title: Localization.returnPolicyTitle, isSubheadline: true, isOn: $viewModel.returnOnNonDelivery)
+                    .padding(.bottom, Constants.verticalPadding)
+                    .padding(.horizontal, Constants.horizontalPadding)
+                    .padding(.horizontal, insets: safeAreaInsets)
+
+                TitleAndValueRow(title: Localization.contentTypeTitle, value: viewModel.contentsType.localizedName, selectable: true) {
+                    showingContentTypes.toggle()
+                }
+                .padding(.horizontal, insets: safeAreaInsets)
+                .sheet(isPresented: $showingContentTypes, content: {
+                    SelectionList(title: Localization.contentTypeTitle,
+                                  items: ShippingLabelCustomsForm.ContentsType.allCases,
+                                  contentKeyPath: \.localizedName,
+                                  selected: $viewModel.contentsType)
+                })
+
+                Divider()
+                    .padding(.leading, Constants.horizontalPadding)
```

Comment:
I think this needs both the constant padding and the safe area insets:

```suggestion
                    .padding(.leading, Constants.horizontalPadding)
                    .padding(.leading, insets: safeAreaInsets)
```

Without the safe area insets, the divider extends too far left in landscape mode:

![Simulator Screen Shot - iPhone 11 Pro - 2021-08-11 at 10 05 55](https://user-images.githubusercontent.com/8658164/129005783-9e330c63-416a-4f71-8006-315ca4248345.png)

Reply:
Thank you! I too have updated this in #4772, please help me check that later 🙇‍♀️